script.js

```diff
@@ -0,0 +1,71 @@
+let defaultResult = 0;
+
+let currentResult = defaultResult;
+
+let logEntries = [];
+
+function getUserInput() {
+  return parseInt(userInput.value);
```

Comment:
```jsx
parseInt(userInput.value);
```
what it is meaning of this line ?

Reply:
User input is always a string to JavaScript, thus to convert them into numbers we use parseInt, which is a built in js function. @anask601